views/container-statuses.html

```diff
@@ -0,0 +1,33 @@
+<div ng-if="containerStatuses.length == 0"><em>none</em></div>
+<dl ng-repeat="containerStatus in containerStatuses | orderBy:'name'" class="dl-horizontal">
+  <dt>Name</dt>
+  <dd>{{containerStatus.name}}</dd>
+  <dt>State</dt>
+  <dd>
+    <span ng-if="containerStatus.state | isEmpty"><em>none</em></span>
+    <span ng-repeat="(state, stateDescription) in containerStatus.state">
```

Comment:
looking at the godocs, state is never supposed to have more than a single child: https://github.com/GoogleCloudPlatform/kubernetes/blob/9aa9260295158de6d885e5daf731dff7c75bd874/pkg/api/v1/types.go#L690-L716

I would limit this loop to 1, and ng-switch on the known state values ("waiting","running","termination", and by default just output the state)
```
<span ng-repeat="(state, stateDescription) in containerStatus.state | limitTo:1">
  <span ng-switch="state">
    <span ng-switch-when="waiting">
      Waiting
      <span ng-if="stateDescription.reason">({{stateDescription.reason}})</span>
    </span>
    <span ng-switch-when="running">
      Started
      <span ng-if="stateDescription.startedAt">at {{stateDescription.startedAt}}</span>
    </span>
    <span ng-switch-when="termination">
      Terminated 
      <span ng-if="stateDescription.finishedAt">at {{stateDescription.finishedAt}}</span>
      <span ng-if="stateDescription.exitCode">with exit code {{stateDescription.exitCode}}</span>
      <span ng-if="stateDescription.reason">({{stateDescription.reason}})</span>
    </span>
    <span ng-switch-default>{{state}}</span>
  </span>
</span>
```
```